examples/seq2seq/seq2seq.py

```diff
@@ -217,7 +221,9 @@ def main():
     optimizer = chainer.optimizers.Adam()
     optimizer.setup(model)
 
-    train_iter = chainer.iterators.SerialIterator(train_data, args.batchsize)
+    train_iter = chainer.iterators.SerialIterator(train_data,
+                                                  args.batchsize,
+                                                  shuffle=False)
```

Comment:
So, can I just remove shuffle=True? 
I fixed it in the latest commit. Please let me know if I'm misunderstanding it and further fixes are necessary.
Thanks!